videos/tasks.py

```diff
@@ -51,6 +56,7 @@ def upload_youtube_videos():
             video_file.destination_id = response["id"]
             video_file.destination_status = response["status"]["uploadStatus"]
             video_file.status = VideoFileStatus.UPLOADED
+            start_transcript_job.delay(video_file.video.id)
```

Comment:
Instead of calling `<task>.delay` from within a task, might be better to do something like this:
```python
group_tasks = []
for video_file in yt_queue:
    ....
    group_tasks.append(start_transcript_job.s(video_file.video.id)

if group_tasks:
    raise self.replace(celery.group(group_tasks))
```